Update Get-ADUser.md (#2145)
MicrosoftDocs/windows-powershell-docs: docset/winserver2012-ps/activedirectory/Get-ADUser.md

Get the user with name 'GlenJohn' on the AD LDS instance.

### -------------------------- EXAMPLE 5 --------------------------
```powershell
C:\PS> Get-ADUser -LDAPFilter '(!userAccountControl:1.2.840.113556.1.4.803:=2)'
```

Description

> -----------

Reviewer: ```suggestion
```
Thank you @doctordns for this contribution!

Let's remove these lines in the examples.

> Get all user accounts in AD that are enabled, using an LDAP filter.

Reviewer 2: ```suggestion
Get all user accounts in Active Directory that are enabled, using an LDAP filter.
```

-Partition "CN=Schema,CN=Configuration,DC=EUROPE,DC=TEST,DC=CONTOSO,DC=COM"

In many cases, a default value is used for the Partition parameter if no value is specified. 
The rules for determining the default value are given below. 

> Note that rules listed first are evaluated first and once a default value can be determined, no further rules is evaluated.

Reviewer: ```suggestion
Note that rules listed first are evaluated first, and once a default value can be determined, no further rules are evaluated.
```

In AD DS environments, a default value for Partition is set in the following cases:  - If the Identity parameter is set to a distinguished name, the default value of Partition is automatically generated from this distinguished name.

- If running cmdlets from an Active Directory provider drive, the default value of Partition is automatically generated from the current path in the drive.

> - If none of the previous cases apply, the default value of Partition is set to the default partition or naming context of the target domain.

Reviewer: ```suggestion
- If none of the previous cases applies, the default value of Partition is set to the default partition or naming context of the target domain.
```

In AD LDS environments, a default value for Partition is set in the following cases:

- If the Identity parameter is set to a distinguished name, the default value of Partition is automatically generated from this distinguished name.
- If running cmdlets from an Active Directory provider drive, the default value of Partition is automatically generated from the current path in the drive.

> - If the target AD LDS instance has a default naming context, the default value of Partition is set to the default naming context.  To specify a default naming context for an AD LDS environment, set the msDS-defaultNamingContext property of the Active Directory directory service agent (DSA) object (nTDSDSA) for the AD LDS instance.

Reviewer 2: ```suggestion
- If the target AD LDS instance has a default naming context, the default value of Partition is set to the default naming context. To specify a default naming context for an AD LDS environment, set the msDS-defaultNamingContext property of the Active Directory directory service agent (DSA) object (nTDSDSA) for the AD LDS instance.
```

> - If none of the previous cases apply, the Partition parameter does not take any default value.

Reviewer: ```suggestion
- If none of the previous cases applies, the Partition parameter does not take any default value.
```
